components/ntp_background_images/browser/ntp_background_images_service.cc

```diff
@@ -73,14 +73,14 @@ std::string HandleComponentData(const base::FilePath& installed_dir) {
   if (json_path.empty()) {
```

Comment:
Super referrals are not related to our changes in the other PR, can we leave these as DVLOG's please.

Reply:
Fixed